openedx/frontend-app-authoring
# feat: enable problem bank button functionality [FC-0062]

## Description

Implements: https://github.com/openedx/frontend-app-authoring/issues/1462

Also, updates library v2 and problem bank icons and adds a beta tag based on api response.

## Supporting information

* Depends on https://github.com/openedx/edx-platform/pull/35802
* https://github.com/openedx/frontend-app-authoring/issues/1462
* `Private-ref`: [FAL-3936](https://tasks.opencraft.com/browse/FAL-3936)

## Testing instructions

* Checkout https://github.com/openedx/edx-platform/pull/35802
* Make sure [librariesV2](https://github.com/open-craft/edx-platform/blob/cb8a1f4976b4f49f3cc77f4ebbab5f5cdb2a2233/cms/djangoapps/contentstore/toggles.py#L659-L669) is enabled.
* Enable new mfe unit page using by setting waffle flag: [contentstore.new_studio_mfe.use_new_unit_page](http://studio.local.openedx.io:8001/admin/waffle/flag/)
* Go to any unit page in any course and verify that `Problem Bank` option is working (adding a problem block). Verify the icons and beta badge for `Problem Bank` and `Library content` buttons.
